components/raftstore/src/store/worker/metrics.rs

```diff
@@ -10,11 +10,18 @@ use tikv_util::time::Instant;
 make_auto_flush_static_metric! {
     pub label_enum SnapType {
        generate,
-       apply,
+       apply
     }
 
+    // snapshot task status
+    // |all---------start--------------|
+    //               |
+    //               |
+    //               V
```

Comment:
what's the meaning of this arrow

Reply:
It means that the trnasfer relation of one snapshot task . It begin at all,  then into  start by executing ,  finally succ. and the abnormal status may be cancel,  abort，fail etc.